RecoTracker/LSTCore/src/alpaka/LSTEvent.h

```diff
@@ -106,26 +112,26 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
                        std::vector<float> const& z,
                        std::vector<unsigned int> const& detId,
                        std::vector<unsigned int> const& idxInNtuple);
-    void addPixelSegmentToEvent(std::vector<unsigned int> const& hitIndices0,
-                                std::vector<unsigned int> const& hitIndices1,
-                                std::vector<unsigned int> const& hitIndices2,
-                                std::vector<unsigned int> const& hitIndices3,
-                                std::vector<float> const& dPhiChange,
-                                std::vector<float> const& ptIn,
-                                std::vector<float> const& ptErr,
-                                std::vector<float> const& px,
-                                std::vector<float> const& py,
-                                std::vector<float> const& pz,
-                                std::vector<float> const& eta,
-                                std::vector<float> const& etaErr,
-                                std::vector<float> const& phi,
-                                std::vector<int> const& charge,
-                                std::vector<unsigned int> const& seedIdx,
-                                std::vector<int> const& superbin,
-                                std::vector<PixelType> const& pixelType,
-                                std::vector<char> const& isQuad);
+    void loadPixelSegmentToEvent(std::vector<float> const& ptIn,
+                                 std::vector<float> const& ptErr,
+                                 std::vector<float> const& px,
+                                 std::vector<float> const& py,
+                                 std::vector<float> const& pz,
+                                 std::vector<float> const& eta,
+                                 std::vector<float> const& etaErr,
+                                 std::vector<float> const& phi,
+                                 std::vector<int> const& charge,
+                                 std::vector<unsigned int> const& seedIdx,
+                                 std::vector<int> const& superbin,
+                                 std::vector<PixelType> const& pixelType,
+                                 std::vector<char> const& isQuad);
 
     void createMiniDoublets();
+    void createPixelSegmentToEvent(std::vector<unsigned int> hitIndices0,
```

Comment:
I'd call this `addPixelSegmentToEventPart2` or `addPixelSegmentToEventFinalize`
otherwise the naming is quite confusing ("add to event" and "create to event")